dom.bs

```diff
@@ -6598,7 +6598,7 @@ dictionary ShadowRootInit {
   SlotAssignmentMode slotAssignment = "named";
   boolean clonable = false;
   boolean serializable = false;
-  CustomElementRegistry customElementRegistry;
+  CustomElementRegistry? customElementRegistry = null;
```

Comment:
Maybe add a comment here explaining why we're departing from our usual preference to only have one way of signaling absence.

Reply:
Yeah, some comment would good, since this is a rather odd case. Wouldn't have guessed anyone using attachShadow that way, but don't see reason why not.